Boolean.CSharp.Main/Bank.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+
+namespace Boolean.CSharp.Main
+{
+    public class Bank: UserAccount
+    {
+        Management management = new Management();
+        public List<Transaction> Transactions { get; set; } = new List<Transaction>();
+        private List<decimal> BalanceHistory { get; set; } = new List<decimal>();
+
+        public void Deposit(string accountName, string accountType,  string branchName, decimal amount)
+        {
+            UserAccount account = UserAccount.GetAccount(accountName, accountType, branchName);
+            if (account != null)
+            {
+                BalanceHistory.Add(amount);
```

Comment:
the BalanceHistory is ambiguous; do you mean the history of the total bank balance over time, or the Log of all amounts?

Reply:
I think looking at the code you mean the second option